src/libs/ajax.js

```diff
@@ -964,7 +958,11 @@ const Workspaces = signal => ({
         const res = await fetchOk(url)
         const payload = await res.json()
 
-        return upsertEntities(payload)
+        const body = _.map(({ name, entityType, attributes }) => {
+          return { name, entityType, operations: attributesUpdateOps(attributes) }
+        }, payload)
+
+        return fetchRawls(`${root}/entities/batchUpsert`, _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }]))
```

Comment:
`importJSON` can be tested with a JSON file such as:
```json
[
  { "entityType": "thing", "name": "thing_1", "attributes": { "value": "foo" } },
  { "entityType": "thing", "name": "thing_2", "attributes": { "value": "bar" } },
  { "entityType": "thing", "name": "thing_3", "attributes": { "value": "baz" } }
]
```

Serve the file with something like `npx http-server . --cors` and navigate to `http://localhost:3000/#import-data?format=entitiesJson&url=http://localhost:8080/things.json`.